crates/blockchain/error.rs

```diff
@@ -58,6 +58,14 @@ pub enum MempoolError {
     BlobTxNoBlobsBundle,
     #[error("Mismatch between blob versioned hashes and blobs bundle content length")]
     BlobsBundleWrongLen,
+    #[error("Nonce for account too low")]
+    InvalidNonce,
+    #[error("Transaction chain id mismatch")]
```

Comment:
Let's see if we can add to this error which is the actual chain-id value that was expected, the [error macro supports this](https://docs.rs/thiserror/latest/thiserror/#details)

Reply:
Good idea, added here [b988558](https://github.com/lambdaclass/lambda_ethereum_rust/pull/1125/commits/b988558443dd54585642e3b720454c2422f56292)